client-templates/nexus/.env.sample

```diff
@@ -0,0 +1,27 @@
+# your unique broker identifier
+BROKER_TOKEN=<broker-token>
+
+# The Base URL for your Nexus Repository Manager
+# If not using basic auth this will only be "https://<your.nexus.hostname>/repository"
+BASE_NEXUS_URL=https://<username>:<password>@<your.nexus.hostname>
+
+# The URL to your Nexus Repository Manager
+NEXUS_URL=$BASE_NEXUS_URL/repository
+
+# The URL of the Snyk broker server
+BROKER_SERVER_URL=https://broker.snyk.io
+
+# the fine detail accept rules that allow Snyk to make API requests to your
+# Nexus instance
+ACCEPT=accept.json
+
+# The path for the broker's internal healthcheck URL. Must start with a '/'.
+BROKER_HEALTHCHECK_PATH=/healthcheck
+
+# Provide RES_BODY_URL_SUB with the URL of the Nexus without credentials and http protocol
+# This URL substitution is required for NPM integration
+# RES_BODY_URL_SUB=http://<your.nexus.hostname>/repository
+
+# Nexus validation url, checked by broker client systemcheck endpoint
+BROKER_CLIENT_VALIDATION_URL=$BASE_NEXUS_URL/service/rest/v1/status
```

Comment:
`/v1/status/check` ?

Reply:
yeah good shout - the `/v1/status/check` requires auth from checking it just now

Reply:
https://support.sonatype.com/hc/en-us/articles/226254487-System-Status-and-Metrics-REST-API

So for a readonly user, i.e. `snyk-readonly` it might not work unless it has those permissions, when it comes to `/v1/status/check`...

Reply:
`curl -v -X 'GET' \
  'https://<pre-prod-nexus-readonly-user>:<pre-prod-nexus-readonly>@nexus.dev.snyk.io/service/rest/v1/status' \      
  -H 'accept: application/json'`
  
  ^ gives 200, whereas:
  
  `curl -v -X 'GET' \
  'https://<pre-prod-nexus-readonly-user>:<pre-prod-nexus-readonly>@nexus.dev.snyk.io/service/rest/v1/status/check' \ 
  -H 'accept: application/json'`
  
  gives 403